src/util.ts

```diff
@@ -357,3 +357,30 @@ export const toCamelCase = (str: string) =>
 
 export const parseBool = (v: string): boolean | string =>
   ({ true: true, false: false })[v] ?? v
+
+// prettier-ignore
+export const parseDotenv = (
+    content: string,
+  ): NodeJS.ProcessEnv => {
+    return content
+      .split(/\r?\n/)
+      .reduce<NodeJS.ProcessEnv>((acc, line) => {
+        const [k,v] = line.split(/(?<!=)=/).map(v => v.trim())
```

Comment:
Lets use split with limit instead. This regex introduces potential vulnerability.

Reply:
d1